packages/frames.js/src/middleware/openframes.ts

```diff
@@ -117,8 +117,11 @@ export function openframes<T = OpenFrameMessage>({
 
     const message = await handler.getFrameMessage(json);
 
-    return next({
-      message,
+    return nextInjectAcceptedClient({
+      nextResult: next({
```

Comment:
this is not defined in context type on purpose?

Reply:
just a helper function that also injects the `accepts` protocol ids into the returned frame definition